man/mergelist.Rd

```diff
@@ -0,0 +1,172 @@
+\name{mergelist}
+\alias{mergelist}
+\title{Merge multiple data.tables}
+\description{
+  Faster merge of multiple \code{data.table}s.
+}
+\usage{
+  mergelist(l, on, cols,
+    how = c("left","inner","full","right","semi","anti","cross"),
+    mult, copy = TRUE,
+    join.many = getOption("datatable.join.many"))
+}
+\arguments{
+  \item{l}{ \code{list} of \code{data.table}s to merge. }
+  \item{on}{ \code{character} of columns to merge on, when missing, then \emph{key} of \emph{join-to} table is used. }
```

Comment:
`on` has to be shared across all inputs right? there's no logic for handling a mapping of column names across tables right?

Reply:
i'm also a bit confused by the wording:

> key of join-to table is used

Reply:
join-to is explained below, `on` will be shared, unless provided as list, then can be specified for each join pair, also explained in details.